routes/v1.js

```diff
@@ -104,4 +109,38 @@ apiRoutes.post(
   UserController.resetPassword
 );
 
+apiRoutes.route('/articles/:slug/comments')
+  .post(
+    Auth.verifyUser,
+    isUserVerified,
+    validateArticleExist,
+    validateCreateComment,
+    returnValidationErrors,
+    CommentController.createComment
+  )
+  .get(
+    Auth.verifyUser,
+    isUserVerified,
+    validateArticleExist,
+    CommentController.getComments
+  );
+
+apiRoutes.route('/articles/:slug/comments/:id')
+  .patch(
```

Comment:
Change your http method to `put`

Reply:
As discussed, `patch` seems to be the more appropriate method for this. Thanks.